pkg/controller/fileobserver/observer_polling.go

```diff
@@ -38,6 +38,12 @@ func (o *pollingObserver) AddReactor(reaction ReactorFn, startingFileContent map
 
 		if startingContent, ok := startingFileContent[f]; ok {
 			klog.V(3).Infof("Starting from specified content for file %q", f)
+			// if the starting content is empty (like file does not exists, do not calculate hash of empty string
```

Comment:
the comment is wrong, it may still be desirable to react to `file does not exists` -> `file exists but is empty`, in this case however it's probably just that the file is empty

Reply:
```
mfojtik 2:45 PM
@stlaz if the file does not exists you don't have starting content

mfojtik 2:46 PM
@stlaz calculateHashFile() handle that
it returns ""
```
looking at the thing as a whole it's not that nice, but good enough for now

Reply:
comment updated.